static-site/main.tf

```diff
@@ -98,39 +97,32 @@ resource "aws_cloudfront_distribution" "sub_domain_distribution" {
   enabled             = true
   default_root_object = "index.html"
 
+  custom_error_response {
+    error_code = "404"
+    response_code = "200"
+    response_page_path = "/404.html"
```

Comment:
We should make the 404 path configurable (variable).  Also, we're going to want to respond 404 instead of 200.

Reply:
Changed back to `200` after I started to see `403`
Related info from aws:
https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/custom-error-pages-response-code.html